README.md

```diff
@@ -94,6 +94,10 @@ Rewrite the prefix to the specified string. Default: `''`.
 
 A `preHandler` to be applied on all routes. Useful for performing actions before the proxy is executed (e.g. check for authentication).
 
+### proxyPayloads
+
+When you set this option as false, you will be able to access the body but it would also disable the direct pass through of the payload. The implementor will have to take care if the payload is proxied correctly.
+
```

Comment:
Could you add an example of usage of this, as it's easy to mess up.

Reply:
yeah will do that

Reply:
Hey,I was thinking... will it be an example like just adding a contentParser for some content-type on fastify or a method actually parsing some data of some content-type?

Reply:
Whatever you want.